docs/man/cdt-cc.1.md

```diff
@@ -0,0 +1,351 @@
+---
+title: CDT-CC
+section: 1
+header: User Manual
```

Comment:
I think this is a better header?
```suggestion
header: Contract Development Toolkit (CDT)
```

Reply:
It looks like cdt-cpp.1.md has the same issue, I think we can fix it here as well.

Reply:
> It looks like cdt-cpp.1.md has the same issue, I think we can fix it here as well.

Updated the headers here and in cdt-cpp